ckanext/datagovuk/public/datagovuk.css

```diff
@@ -78,3 +79,41 @@ a.btn .icon-briefcase {
 label:after {
   content: "";
 }
+
+@media (min-width: 768px) {
+  .hero {
+    background: #128400;
```

Comment:
The hero block is a different colour on smaller screens?

Reply:
Complete oversight!  Previously the tile pattern only showed if the screen was over that width.  We want the colour to be the same all the time.